### DISCUSSION TITLE
How should I obtain it “winChecksumSeed & macChecksumSeed”?

### DISCUSSION THREAD

**Discussion by PkllonG** [azerothcore/azerothcore-wotlk#17622] [General]
https://github.com/azerothcore/azerothcore-wotlk/issues/16932
StrictVersionCheck = 1
Unable to log in after use.

**Comment by PkllonG** [comment]
https://github.com/vmangos/LoginHashCalculator

**Reply by qyh214** [reply]
how to use it?

**Comment by Maypul** [comment]
I'd like to really know how to properly use it @PkllonG . I have managed to generate hash, but it is not working when I update the value in the database. Even generating hash for vanilla unmodified .exe gives different string than the one that is in the database.

**Comment by PkllonG** [comment]
I also did not succeed
